dts/msm8226/rules.mk

```diff
@@ -4,5 +4,9 @@ DTBS += \
 	$(LOCAL_DIR)/apq8026-asus-sparrow.dtb \
 	$(LOCAL_DIR)/apq8026-huawei-sturgeon.dtb \
 	$(LOCAL_DIR)/apq8026-lg-lenok.dtb \
+	$(LOCAL_DIR)/apq8026-samsung-r00.dtb \
+	$(LOCAL_DIR)/apq8026-samsung-r01.dtb \
+	$(LOCAL_DIR)/apq8026-samsung-r02.dtb \
+	$(LOCAL_DIR)/apq8026-samsung-r03.dtb \
```

Comment:
Are you sure you need all of them? Usually the device's bootloader will only select the last one.

Reply:
Not sure really, I just added the ones which are in downstream kernel

Reply:
I now removed the revisions 0-2 and only kept revision 3.